src/api_clients/athena.py

```diff
@@ -0,0 +1,44 @@
+import boto3
+import pandas as pd
+from time import sleep
+from prefect import task
+from utils import config
+
+@task()
+def AthenaQuery(query: str = None):
```

Comment:
- :snake: Python functions are styled using snake case
- `query` should probably be a required attribute
```suggestion
def athena_query(query: str):
```